components.js

```diff
@@ -44,7 +44,12 @@ var components = {
 		},
 		"markup": {
 			"title": "Markup",
-			"aliasTitles": { "html": "HTML", "xml": "XML", "svg": "SVG", "mathml": "MathML" },
+			"aliasTitles": {
+				"html": "HTML",
+				"xml": "XML",
+				"svg": "SVG",
+				"mathml": "MathML"
+			},
```

Comment:
Lots of changes in this file were made by prettier in my editor. I tired to undo most of them, but I left in these last ones that I think make the file more consistent. Let me know though if you disagree and I can fi these as well